rlog/2023-08-04-wakurtosis-waku-scallability.mdx

```diff
@@ -0,0 +1,174 @@
+&mdash;
+layout: post
+name: 'Scaling the Waku Protocol: A Performance Analysis with Wakurtosis'  
+title: 'Scaling the Waku Protocol: A Performance Analysis with Wakurtosis'
+date: 2023-10-19 12:00:00
+authors: Daimakaimura
+published: true
+slug: wakurtosis-waku-scallability-simulations
+categories: waku, wakurtosis
+&mdash;
+
+# Scaling the Waku Protocol: A Performance Analysis with Wakurtosis
+
+## Introduction
+
+[Waku](https://waku.org/) is a family of P2P protocols enabling private, metadata-resistant messaging for Web3 by providing censorship resistance, adaptability, modular design, and a shared service network. 
```

Comment:
```suggestion
[Waku](https://waku.org/) is a family of P2P protocols enabling private, scalable messaging for Web3 by providing censorship resistance, adaptability, modular design, and a shared service network. 
```

Not sure what _metadata-resistant_ means. Do you mean _resistant to metadata harvesting_?